other/edge-server/client/README.md

```diff
@@ -1,65 +1,61 @@
-   get Edge Server enabled for your new app.
-2. Download, configure, and start the Edge Server.
-3. Install dependencies and start the Node.js Express server.
-4. Install dependencies and start the React server.
-
 This is a MERN stack that connects to the Edge Server instead of Atlas.
 In this version of the example app, MongoDB wire protocol connections connect
-directly to Edge Server without authorization.
+directly to Edge Server. You can use email/password authentication with an
+App Services user or bypass authentication altogether.
 
 For a MERN stack tutorial, refer to
 [How to Use Mern Stack: A Complete Guide](https://www.mongodb.com/languages/mern-stack-tutorial).
 
-For more information about Edge Server, refer to the
-[Edge Server documentation](https://www.mongodb.com/docs/atlas/app-services/edge-server/).
-
-Atlas Edge Server is currently in Private Preview. To learn more about
-previewing Edge Server, refer to the product page at
-[Edge Server](https://www.mongodb.com/products/platform/atlas-edge-server).
-
-## Project npm commands
-
-- `install-deps`: installs the dependencies for the Express server and
-  React client. You must install the Edge Server dependencies separately.
-- `start`: starts the Edge Server, Express server, and React client.
-- `shut-down`: shuts down the Edge Server, Express server, and React client.
+Atlas Edge Server is currently in Public Preview. To learn more about Edge Server,
+refer to the product page at [Edge Server](https://www.mongodb.com/products/platform/atlas-edge-server)
+and the [Edge Server documentation](https://www.mongodb.com/docs/atlas/app-services/edge-server/).
 
 ## Quick Start
 
-1. Create an App Services App based on the `edge-server.todo` template app, and
-   get Edge Server enabled for your new app.
-2. Download and configure the Edge Server.
-3. Start Docker
-4. From the project's root directory, run `npm run install-deps`.
-5. Add a `.env` file inside the `node-server` directory with the details
+1. Create an App Services App based on the `edge-server.todo` template app. Refer
+   to [the documentation](https://www.mongodb.com/docs/atlas/app-services/reference/template-apps/#create-a-template-app) for details.
+2. [Create a Service for Edge Server](https://www.mongodb.com/docs/atlas/app-services/edge-server/manage-edge-servers/#create-a-service-for-edge-server)
+3. [Create an Edge Server Instance](https://www.mongodb.com/docs/atlas/app-services/edge-server/manage-edge-servers/#create-an-edge-server-instance)
+4. [Install and Configure the Edge Server Instance](https://www.mongodb.com/docs/atlas/app-services/edge-server/configure/#install-and-configure-the-edge-server-instance)
+5. As part of the Edge Server installation, choose to create a new user. Remember
+   the user's email and password for later.
+   Alternatively, you can skip user creation and disable authentication by running `edgectl config --insecure-disable-auth=true`.
+6. Start Docker
```

Comment:
If you want to keep "Start Docker" in here, I'd move it before the "Install and Configure the Edge Server Instance" step. When I was just running through this with Lindsey, we found the `edgectl init` step expects Docker to be running already.

Reply:
Ah, right. I do want to selfishly keep it. Multiple times I've forgotten to start Docker. I know the CLI will tell people they need to do it, but I think it helps to have it in the steps.